src/components/AnonymousChat/ChatDetailHeader.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import FastImage from 'react-native-fast-image';
+import IconEP from 'react-native-vector-icons/Entypo';
+import {StyleSheet, Text, View} from 'react-native';
+
+import AnonymousIcon from '../../screens/ChannelListScreen/elements/components/AnonymousIcon';
+import ChannelImage from '../ChatList/elements/ChannelImage';
+import CustomPressable from '../CustomPressable';
+import IcArrowBackWhite from '../../assets/arrow/Ic_arrow_back_white';
+import dimen from '../../utils/dimen';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
+import {colors} from '../../utils/colors';
+import {fonts} from '../../utils/fonts';
+import {SIGNED} from '../../hooks/core/constant';
+
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'row',
+    height: 50,
+    alignItems: 'center'
+  },
+  bgDarkBlue: {
+    backgroundColor: colors.darkBlue
+  },
+  bgBondiBlue: {
+    backgroundColor: colors.bondi_blue
+  },
+  backButton: {
+    paddingLeft: 22,
+    paddingRight: 20,
+    height: 50,
+    justifyContent: 'center'
+  },
+  avatar: {
+    width: dimen.normalizeDimen(38),
+    height: dimen.normalizeDimen(38),
+    borderRadius: dimen.normalizeDimen(38),
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  textContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    flex: 1,
+    alignItems: 'center'
+  },
+  text: {
+    fontFamily: fonts.inter[600],
+    color: colors.white,
+    fontSize: 14,
+    lineHeight: 17,
+    marginLeft: 10,
+    flex: 1
+  },
+  threeDot: {
+    paddingRight: 22,
+    paddingLeft: 22,
+    justifyContent: 'center',
+    height: 50
+  },
+  avatarImage: {
+    height: dimen.normalizeDimen(40),
+    width: dimen.normalizeDimen(40),
+    borderRadius: dimen.normalizeDimen(20)
+  }
+});
+
+const ChatDetailHeader = ({
+  avatar = DEFAULT_PROFILE_PIC_PATH,
+  user,
+  onBackPress = () => console.log('onBackPress'),
+  onThreeDotPress = () => console.log('onThreeDotPress'),
+  onAvatarPress = () => console.log('onAvatarPress'),
+  anon_user_info_emoji_code,
+  anon_user_info_color_code,
+  type,
+  channel
+}) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop types for this component are not defined. It's recommended to use PropTypes or TypeScript to define the types of the props. This will help catch bugs related to incorrect prop types and improve code readability.

```javascript
ChatDetailHeader.propTypes = {
  avatar: PropTypes.string,
  user: PropTypes.string.isRequired,
  onBackPress: PropTypes.func,
  onThreeDotPress: PropTypes.func,
  onAvatarPress: PropTypes.func,
  anon_user_info_emoji_code: PropTypes.string,
  anon_user_info_color_code: PropTypes.string,
  type: PropTypes.string.isRequired,
  channel: PropTypes.object,
};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->